vendor/brave-ios/Ledger/Generated/NativeLedgerClient.mm

```diff
@@ -237,4 +230,28 @@
 }
 void NativeLedgerClient::GetFirstContributionQueue(ledger::GetFirstContributionQueueCallback callback) {
   return [bridge_ getFirstContributionQueue:callback];
-}
\ No newline at end of file
+}
+void NativeLedgerClient::InsertOrUpdatePromotion(ledger::PromotionPtr info, ledger::ResultCallback callback) {
+  return [bridge_ insertOrUpdatePromotion:std::move(info) callback:callback];
+}
+void NativeLedgerClient::GetPromotion(const std::string& id, ledger::GetPromotionCallback callback) {
+  return [bridge_ getPromotion:id callback:callback];
+}
+void NativeLedgerClient::InsertOrUpdateUnblindedToken(ledger::UnblindedTokenPtr info, ledger::ResultCallback callback) {
+  return [bridge_ insertOrUpdateUnblindedToken:std::move(info) callback:callback];
+}
+void NativeLedgerClient::GetAllUnblindedTokens(ledger::GetAllUnblindedTokensCallback callback) {
+  return [bridge_ getAllUnblindedTokens:callback];
+}
+void NativeLedgerClient::DeleteUnblindedToken(const std::vector<std::string>& id_list, ledger::ResultCallback callback) {
+  [bridge_ deleteUnblindedToken:id_list callback:callback];
+}
+ledger::ClientInfoPtr NativeLedgerClient::GetClientInfo() {
+  return [bridge_ getClientInfo];
+}
+void NativeLedgerClient::UnblindedTokensReady() {
+  [bridge_ unblindedTokensReady];
+}
+void NativeLedgerClient::GetAllPromotions(ledger::GetAllPromotionsCallback callback) {
+  [bridge_ getAllPromotions:callback];
+}
```

Comment:
cc @kylehickinson

Reply:
@masparrow Not sure what you mean. Can you elaborate?

Reply:
@kylehickinson sure; 

`void NativeLedgerClient::GetAllPromotions(ledger::GetAllPromotionsCallback callback) const {`

If it's a member method that doesn't change the actual object. Again, forgive me if I've missed the obvious, my eyes were crossing during review...

Reply:
Ah, this would mean a change on the definition then in `ledger_client.h`, and I actually think `const` callbacks caused issues if I recall correctly which is why they were all switched. @NejcZdovc can confirm

Reply:
yes there were problem with const callbacks